functions/src/definitions/userHandlers.js

```diff
@@ -83,6 +73,41 @@ exports.userHandlerWhatsapp = async function (message) {
         }
         break
       }
+      if (
+        normalizeSpaces(message.text.body) //normalise spaces needed cos of potential &nbsp when copying message on desktop whatsapp
+          .toLowerCase()
+          .startsWith(
+            "welcome to checkmate! send in this message to get started, and credit your friend with your referral. code:"
```

Comment:
May want to throw the original string to `constants.ts` and lowercase it again for comparison

Reply:
done